esmf_regrid/experimental/io.py

```diff
@@ -52,11 +52,18 @@ def save_regridder(rg, filename):
     regridder_type = rg.__class__.__name__
     if regridder_type == "GridToMeshESMFRegridder":
         src_grid = (rg.grid_y, rg.grid_x)
-        src_shape = [len(coord.points) for coord in src_grid]
+        if len(src_grid[0].shape) == 1:
+            src_shape = [len(coord.points) for coord in src_grid]
```

Comment:
Using `len(coord.points)` is slightly misleading as it gives the length of the first dimension, rather than the number of items in the array. Given that we've checked that `rg.grid_y` is 1D it doesn't matter here, but `coord.points.size` is equivalent and feels clearer.